app/routes/projects/index.tsx

```diff
@@ -90,20 +91,23 @@ export default function Projects() {
     return firstName.substring(0, 1) + lastName.substring(0, 1)
   }
 
-  const deleteFilter = (filter: string, value: string | null) => {
-    searchParams.delete(filter)
-    setSearchParams(searchParams)
+  const deleteFilterUrl = (filter: string, value: string | null) => {
+    const newParams = new URLSearchParams(searchParams.toString());
```

Comment:
We need to duplicate the `searchParams` using `new URLSearchParams` to avoid modifying the object for other links.